### PR TITLE
Spatel/net 1646 add max ejection percent and base ejection time

### PR DESCRIPTION
**Pull request by malizz**
Changes proposed in this PR:
add max ejection percent and base ejection time to passiveHealthCheck 

How I've tested this PR: local test and with wan federation setup 

How I expect reviewers to test this PR:

Checklist:
- [x] Tests added
- [ ] CHANGELOG entry added 
  > HashiCorp engineers only, community PRs should not add a changelog entry.
  > Entries should use present tense (e.g. Add support for...)

### CODE REVIEW

File: control-plane/api/v1alpha1/servicedefaults_types.go [hunk 1]
```diff
@@ -5,20 +5,18 @@ package v1alpha1
...
 	"github.com/miekg/dns"
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/util/validation/field"
-
-	"github.com/hashicorp/consul-k8s/control-plane/api/common"
+	"net"
+	"strings"
```
Comment (thisisnotashwin) [inline control-plane/api/v1alpha1/servicedefaults_types.go:19]: nit: the import format needs to get fixed.

**Review by thisisnotashwin — COMMENTED** [review]
This is looking great @malizz but i seem to get a diff in the `zz_generated_deepcopy` when i run `make ctrl-manifests ctrl-generate`. Could you run that again?

File: control-plane/api/v1alpha1/servicedefaults_types.go (hunk 1 above)
Comment (malizz) [reply control-plane/api/v1alpha1/servicedefaults_types.go]: thanks for catching that, fixed now.

**Review by DanStough — APPROVED** [review]
LGTM @malizz !

**Pull request merged**